packages/graph-explorer/src/workspaces/Settings/LoadConfigButton.tsx

```diff
@@ -94,77 +96,73 @@ function ConfirmationModal({
   onConfirm: () => void;
   isPending: boolean;
 }) {
-  const [_, { close }] = useDisclosure(Boolean(backupData));
-
   // Used to ensure the file name does not disappear while the modal is animating out
   const debouncedFileName = useDebounceValue(fileName, 200);
 
-  const internalOnCancel = () => {
-    onCancel();
-    close();
-  };
```

Comment:
I tested through the success, cancel, and error states and this seems to work fine.